internal/templating/ingress/templates_ingress.go

```diff
@@ -100,26 +100,16 @@ func GenerateIngressTemplate(
 	}
 	if lValues.EnvironmentType == "production" {
 		// monitoring is only available in production environments
-		additionalAnnotations["monitor.stakater.com/enabled"] = "false"
 		primaryIngress, _ := url.Parse(lValues.Route)
 		// check if monitoring enabled, route isn't autogenerated, and the primary ingress from the .lagoon.yml is this processed routedomain
 		// and enable monitoring on the primary ingress only.
 		if lValues.Monitoring.Enabled && !route.Autogenerated && primaryIngress.Host == route.Domain {
-			additionalLabels["lagoon.sh/primaryIngress"] = "true"
-
-			// only add the monitring annotations if monitoring is enabled
-			additionalAnnotations["monitor.stakater.com/enabled"] = "true"
-			additionalAnnotations["uptimerobot.monitor.stakater.com/alert-contacts"] = "unconfigured"
-			if lValues.Monitoring.AlertContact != "" {
-				additionalAnnotations["uptimerobot.monitor.stakater.com/alert-contacts"] = lValues.Monitoring.AlertContact
-			}
-			if lValues.Monitoring.StatusPageID != "" {
-				additionalAnnotations["uptimerobot.monitor.stakater.com/status-pages"] = lValues.Monitoring.StatusPageID
-			}
-			additionalAnnotations["uptimerobot.monitor.stakater.com/interval"] = "60"
+			additionalLabels["route.lagoon.sh/primary"] = "true"
```

Comment:
```suggestion
			additionalLabels["lagoon.sh/primaryIngress"] = "true"
```
As we already use this annotation and it is set in many many ingresses I think it would be cool to keep it. If we want to change the annotation we will need to have a way to change the already deployed ingresses and also change the ingress operator